src/ansible_runner/display_callback/callback/awx_display.py

```diff
@@ -70,7 +70,7 @@
 
 
 def current_time():
-    return datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc)
+    return datetime.datetime.now(datetime.UTC)
```

Comment:
```suggestion
    return datetime.datetime.now(datetime.timezone.utc)
```

Alias `datetime.UTC` is added in Python 3.11